README.md

```diff
@@ -4,26 +4,26 @@
 [![docker](https://img.shields.io/docker/v/ostis/sc-machine?arch=amd64&label=Docker&logo=Docker&sort=date)](https://hub.docker.com/r/ostis/sc-machine)
 
-<b>SC-machine</b> is a software implementation of semantic network storage and processing of information in language
-of universal knowledge representation SC-code, provided by [<b>the OSTIS Technology</b>](https://github.com/ostis-ai/ostis-standard). 
-SC-machine is a <u>platform-independent graph database management system</u> to store, retrieves and run tasks (agents) 
+**SC-machine** is a software implementation of semantic network storage and processing of information in language
+of universal knowledge representation SC-code, provided by [**the OSTIS Technology**](https://github.com/ostis-ai/ostis-standard). 
+SC-machine is a **platform-independent graph database management system** to store, retrieve and run tasks (agents) 
 on knowledge graphs. All declarative (data, data structures, documentation, tasks specification, etc.) and procedural 
-(programs, modules, systems, communication between systems) knowledge represents on one language, named as SC-code. 
+(programs, modules, systems, communication between systems) knowledge represents on the same language, named SC-code. 
 Nowadays sc-machine includes:
-- semantic storage to store graph constructions, named as sc-memory (graph database);
-- event-based agents processing, named as sc-event-manager (asynced programs workflows handling);
+- semantic storage to store graph constructions, named sc-memory (graph database);
+- event-based agents processing, named sc-event-manager (asynced program workflows handling);
 - tools to communicate with storage:
-   - <u>native</u> program API on C++;
-   - <u>network</u> program API, provided by <i>sc-server</i> to communicate with storage by network with help of Websocket 
+   - *native* API on C++;
+   - *network* API, provided by *sc-server* to communicate with storage by network with help of Websocket 
      and JSON languages;
-   - <i>sc-builder</i> to load information from SCs files into storage (SCs-code is a one of external representation 
+   - *sc-builder* to load information from SCs files into storage (SCs-code is a one of external representation 
      languages of SC-code);
 
 SC-machine provide complete and sufficient native and network APIs to create, update, delete and retrieve operations and
 also supports event-driven communication with sc-memory and sc-agents, located in it.
 
-Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions <u>by the 
-way of one technology</u>, <u>one data representation and programming language</u> and <u>one program software</u>!
+Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions ***by the 
+way of one technology***, ***one data representation and programming language*** and ***one program software***!
```

Comment:
I am not sure that "one" is correct word here. Maybe "the same" or smth like that